apps/web/src/components/WalletConnection/ConnectWalletButton.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable */
+import React, { useState } from 'react'
+import { useWeb3React } from '@web3-react/core'
+import { InjectedConnector } from '@web3-react/injected-connector'
+import { BoxLayout, Text, StackLayout } from '@bloxifi/ui'
+
+export const ConnectWalletButton = () => {
+  const [error, setError] = useState(false)
+  const { activate, active, account, deactivate } = useWeb3React()
```

Comment:
maybe rename active to `active: isConnected` if that has more meaning or even better leave it like this
web3Context = useWeb3React()

and then use it `if(web3Context.active)`